Lexplorer/Program.cs

```diff
@@ -16,6 +16,10 @@
 builder.Services.AddSingleton<EthereumService>();
 builder.Services.AddSingleton<NftMetadataService>();
 builder.Services.AddLazyCache();
+builder.Services.AddHttpClient<ILoopStatsService, LoopStatsService>(client =>
+{
+    client.BaseAddress = new Uri(builder.Configuration.GetSection("services:loopstats:endpoint").Value);
+});
```

Comment:
I like getting the Endpoint from the appsettings - we should consider this for the other services too, but separately...

Reply:
Yeah I've seen it and as I understand that's why you needed Microsoft.Extensions.Configuration added to the project and the using. Looks good to me.